tests/unit/Config.js

```diff
@@ -470,7 +470,7 @@ describe('Config', () => {
             process.env.S3_CONFIG_FILE = oldConfig;
         });
 
-        it('should set up utapi local cache', () => {
+        it.skip('should set up utapi local cache', () => {
```

Comment:
this should make linter fail : as it does not, it seems something is missing from the config :-(
so we'll need to add it later...